pkgs/development/python-modules/kalshi-python/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  urllib3,
+  certifi,
+  python-dateutil,
+  six,
+}:
+
+buildPythonPackage rec {
+  pname = "kalshi-python";
+  version = "2.0.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "kalshi_python";
+    hash = "sha256-ybO7O+rxS3rSo6GN/FZC/BhSnlfH5/+TpJkSxhRBYYw=";
+  };
+
+  dependencies = [
+    urllib3
+    certifi
+    python-dateutil
+    six
+  ];
+
+  build-system = [ setuptools ];
+
```

Comment:
Could you enable tests with `pytestCheckHook` and `pythonImportsCheck`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
I'll look into it, I suspect they will fail in the sandbox as they are testing against the API tho